apps/mdn/mdn-aws/k8s/Makefile

```
@@ -267,14 +272,14 @@ test-k8s-deployments: test-k8s-kuma-deployments test-k8s-kumascript-deployments
 k8s-delete-deployments: k8s-delete-kuma-deployments \
 				k8s-delete-kumascript-deployments
 
-k8s-kuma-deployments: k8s-web k8s-api k8s-celery k8s-ssr
+k8s-kuma-deployments: k8s-web k8s-api k8s-celery k8s-ssr k8s-hpa-web k8s-hpa-api k8s-hpa-celery-workers k8s-hpa-ssr
 
 test-k8s-kuma-deployments: test-k8s-web test-k8s-api test-k8s-celery test-k8s-ssr
```

Does it make sense to add the HPA test targets to this?
```suggestion
test-k8s-kuma-deployments: test-k8s-web test-k8s-api test-k8s-celery test-k8s-ssr    test-k8s-hpa-web test-k8s-hpa-api test-k8s-hpa-celery-workers test-k8s-hpa-ssr
```